clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -490,6 +500,24 @@ public void ingestSSTFiles(RocksDB rocksDB, List<ColumnFamilyHandle> columnFamil
     }
   }
 
+  public void createSnapshot(RocksDB rocksDB) {
+    try {
+      Checkpoint checkpoint = makeCheckpoint(rocksDB);
+      String checkpointPath = this.fullPathForPartitionDBSnapshot;
```

Comment:
The value doesn't change so do we have to re-assign the variables? Is it for making the codes shorter?